src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,72 @@
 import React from 'react';
 import './CurrentUser.scss';
+import { User } from '../../Types/User';
+import { getUser } from '../../API/api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  UserId: number,
```

Comment:
Use proper formatting
```suggestion
  userId: number,
```